excel_import_export/security/ir.model.access.csv

```diff
@@ -0,0 +1,4 @@
+"id","name","model_id:id","group_id:id","perm_read","perm_write","perm_create","perm_unlink"
+xlsx_template_user,xlsx_template_user,model_xlsx_template,,1,1,1,1
```

Comment:
should not you have 2 levels? manager to make the set up and users to be able to import/export?

Reply:
I think we now aim for developer to use first. But if for some import/export action, I think developer can add groups security as they wish (per normal way).
But if in practical, there is need, we can add in next iteration.